### ISSUE TITLE
Range generators in tutorial need fix

### ISSUE THREAD

**Issue by jipphysics** [JuliaGraphs/GraphPlot.jl#114]
When I run the examples in the tutorial on Julia

```
using LightGraphs
nodelabel = [1:nv(g)]
gplot(g, nodelabel=nodelabel)
```

I get the error

```
Must have one label per node (or none)
 ...
```

The error is fixed if you insert a semicolon in the range generator

```
using LightGraphs
nodelabel = [1:nv(g);]
gplot(g, nodelabel=nodelabel)
```

My version of Julia is

```
Julia Version 1.0.3
Commit 099e826241 (2018-12-18 01:34 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.0 (ORCJIT, skylake)
```

**Comment by simonschoelly** [comment]
Which tutorial are you talking about?

I think it should also be possible to just write
```julia
nodelabel = 1:nv(g)
```
or

```julia
nodelabel = vertices(g)
```

**Comment by etiennedeg** [comment]
fixed by #169 I think
